src/App.js

```diff
@@ -0,0 +1,57 @@
+import './App.css';
+import React from 'react';
+import {
+  BrowserRouter, Routes, Route, useLocation,
+} from 'react-router-dom';
+import axios from 'axios';
+// import Root from './root/Root';
+import MainPage from './components/MainPage';
+// import ErrorPage from './components/ErrorPage';
+import DetailsPage from './components/DetailsPage';
+import ToDeleteList from './components/ListToDelete';
+import 'bootstrap-icons/font/bootstrap-icons.css';
+import Login from './root/Login';
+import Signup from './root/CreateAccount';
+import Private from './components/Private';
+import Navigation from './components/Navigation';
+import AddCar from './components/AddCar';
+import ReserveCar from './components/ReserveCar';
+import Reservations from './components/Reservations';
+
+const App = () => {
```

Comment:
- [ ] The blank page is gone, however, when I try to login with a registered user, whether is the included "user1" or my own previously created, this happens:

![image](https://github.com/devalibello/final-capstone-project/assets/57421823/5a18c5a3-a8e3-4dd1-a1cb-ca4fae86a01d)


Kindly, fix this bug so your app works as intended.

_tip:  Remove your local storage or test the project on an incognito window in order to be able to debug this effectively._

Reply:
- Refactored `sessions_controller` to resolve bug.